run_project_tests.py

```diff
@@ -636,10 +636,29 @@ def check_file(fname):
         linenum += 1
 
 def check_format():
+    check_suffixes = {'.c',
+                      '.cpp',
+                      '.cxx',
+                      '.cc',
+                      '.rs',
+                      '.f90',
+                      '.vala',
+                      '.d',
+                      '.s',
+                      '.m',
+                      '.mm',
+                      '.asm',
+                      '.java',
+                      '.txt',
+                      '.py',
+                      '.swift',
+                    }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [Sider](https://sider.review) -->